README.md

```diff
@@ -40,95 +42,88 @@ With CameraKit you are able to seamlessly do the following...
 - Built-in tap to focus.
 - Built-in pinch to zoom.
 
+## Sponsored By
+<a href="https://www.infitting.com/"><img src=".repo/gh-brand-infitting-logo.svg" height="50px" style="margin: 20px"></a>
+<a href="https://www.expensify.com/"><img src=".repo/gh-readme-expensify-logo.svg" height="50px" style="margin: 20px"></a>
+<a href="https://www.goosechase.com/"><img src=".repo/gh-brand-goosechase-logo.svg" height="50px" style="margin: 20px"></a>
+<a href="https://www.alpha-apps.ae/"><img src=".repo/gh-brand-alphaapps-logo.png" height="50px" style="margin: 20px; margin-bottom: 25px"></a>
+<a href="https://www.buddy.works/"><img src=".repo/gh-readme-buddyworks.png" height="100px"></a>
 
-## Setup
-Add __CameraKit__ to the dependencies block in your `app` level `build.gradle`:
+# Pardon the dust
+__CameraKit__ has been going through major changes in the past few months. We've slowed support on verison `0.13.2` and moved our focus to `1.0.0`. This release will bring improved stability, faster processing and a standard API for future development of __CameraKit__. The lastest version is currently in beta, `1.0.0-beta3.9`. 
 
-```groovy		
-compile 'com.wonderkiln:camerakit:0.13.1'
-```		
+The code lives on the branch `v1.0.0`. You can read the discussion about `1.0.0` on the pull request #318. If you have a question or want to talk directly with the team, leave us a message on [spectrum.chat](https://spectrum.chat/camerakit/).
 
+The official `1.0.0` release is coming in a few short weeks. With it we will launch an all new documentation site. In the mean time check out the setup instructions for `1.0.0-beta3.9` below!
 
-## Usage
 
-To use CameraKit, simply add a `CameraView` to your layout:
+## Setup
+To include __CameraKit__ in your project, add the following to your `app` level `build.gradle`.
+```
+repositories {
+    maven { url 'https://dl.bintray.com/camerakit/camerakit-android' }
```

Comment:
I don't have much experience, but in my testing it seems like all that is needed in android > build.gradle

```
repositories {
    jcenter()
}
```
It fetches https://jcenter.bintray.com/com/camerakit/camerakit/1.0.0-beta3.9/camerakit-1.0.0-beta3.9.pom for me without the maven url in repositories.

Additionally, https://dl.bintray.com/camerakit/camerakit-android does not resolve to anything.

Reply:
A little more looking around found me the repo is http://dl.bintray.com/camerakit/camerakit-v1-prereleases